Auth/Logging.pm

```diff
@@ -58,7 +58,8 @@ sub log_incopyright_access  {
             if ( 
                 Debug::DUtils::DEBUG('super') || 
                 Debug::DUtils::DEBUG('supercalifragilisticexpialidocious') || 
-                ( ref($auth) && $auth->user_has_total_access($C) )
+                ( ref($auth) && $auth->user_has_total_access($C) ) || 
+                ( ref($auth) && $auth->user_is_print_disabled_proxy($C) )
```

Comment:
Is the new HT_STAFF_USER stuff interacting with the proxy stuff in ways I'm not seeing, or is this line just fixing a previous oversight?

Reply:
Yep: proxies _do not_ have `access=total` and were not setting this header, so the plack middleware couldn't see this information.